### DISCUSSION TITLE
2007/10/18/b%E9%A9%BE%E7%85%A7%E7%9A%84%E7%83%A6%E6%81%BC

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#183] [Announcements]
# 2007/10/18/b%E9%A9%BE%E7%85%A7%E7%9A%84%E7%83%A6%E6%81%BC

去年驾照换证的时候，是别人帮我办的，他把我的驾照从C照升级为了B1。换完照，到网上一查才知道，变成B照是很讨厌的一件事。B1照比C1照多了驾驶20坐一下中型客车的许可，估计我不会有机会开客车的。

https://blog.qizhen.xyz/2007/10/18/b%E9%A9%BE%E7%85%A7%E7%9A%84%E7%83%A6%E6%81%BC

**Comment by ruanqizhen** [comment]
之前博客下的留言：

涵慈
2007年10月24日 21:33 编辑
你老师傅咯~~那像偶啊~~555,国庆,偶居然把偶自己省吃俭用的新车的挡泥板给刮裂了~~555,偶新手~~C1还是赶上考试改革前考到的！！！如果按现在的考试标准偶根本过不了！！！
对咯~~是哦~~偶也想问问你干嘛升B照啊？

回复
杰
2007年10月22日 19:36 编辑
车管所都有体检的地方，这几乎是常识。呵呵，他们也要创收啊。
我是B照，明年换证，能换成A了，哈，我喜欢。一年体检一次也不麻烦。
下面如果换车子，我就换成50座的金龙！

回复
Nicholas
2007年10月20日 13:27 编辑
不晓得你到底是舍不得那破B1还是不知道怎么降级到C1。偶3年前换新照的时候，就直接填了个自动降级要求，又没得开车开，也基本不会有任务要开9座以上的中巴，干嘛要弄个B1来麻烦自己呢？C1多好，6年才审一次！

回复
YY
2007年10月18日 21:02 编辑
看来驾驶水平不低哦：）

回复
fang
2007年10月18日 18:45 编辑
那你为什么要升级为b照呢？

回复
